packages/openapi-spec-builder/src/openapi-spec-builder.ts

```diff
@@ -162,4 +162,16 @@ export class OperationSpecBuilder extends BuilderBase<OperationObject> {
   withOperationName(name: string): this {
     return this.withExtension('x-operation-name', name);
   }
+
+  /**
+   * Describe tags associated with the operation
+   * @param tags
+   */
+  withTags(tags: string | string[]): this {
```

Comment:
The spec level tags should be an array of [tagObject](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#tagObject).

Reply:
@jannyHou Got it. I didn't realize that the method is for the operation spec builder.